internal/trigger/messagebus/messagebus.go

```diff
@@ -17,25 +17,69 @@
 package messagebus
```

Comment:
Same comment above w/r/t stutter -- messagebus/messagebus.go

Reply:
I fixed the runtime one, however with these triggers in particular I'd like to leave them as is. The idea is that each folder represents the pieces needed to implement the trigger interface. It is primarily for organization in folder structure. I took a stab at removing the folder structure just so that they are at the trigger level, however that creates cyclical imports due to the trigger implementations requiring the context which in turn requires the trigger interface/package. So, depending on how strongly you feel about a filepath stutter, i'm inclined to let these folders stutter since I'd envision that triggers will grow and we will want to keep each one isolated and the organization would be more beneficial.
![image](https://user-images.githubusercontent.com/22204562/54848901-caf3d800-4c9f-11e9-8f6c-d9b69c4f8fdd.png)

Reply:
OK, so you've touched on a problem there. We have a /pkg package which is available to external applications and inside /pkg, we refer to /internal packages. Prior to modules if a 3rd party application (outside of this repo) imported /pkg -- which I believe is the intent -- the build would fail with these imports as the compiler would not traverse into the vendored /internal package. Even now if someone were to directly reference the internal package from a 3rd party app in the following manner
```
import "github.com/edgexfoundry/app-functions-sdk-go/internal"

func main() {
	timeout := internal.BootTimeoutDefault
}
```
They will get an error. Facilitating a consumer to go through /pkg to get to /internal seems to break encapsulation. 

Notice that the Device SDK does not do this, nor did `edgex-go` which made it very easy for us to split `/pkg` out into the `go-mod-core-contracts` module. It seems like a desirable pattern would be to inject resources into the sdk core rather than coupling to them.

Another example toward best practices is go look at Docker's [pkg](https://github.com/moby/moby/tree/master/pkg) directory. If you clone their repo and then look for "/internal" there aren't any references. 
```
[tconn@Borg9 14:15]
~/Documents/code/git/moby/moby/pkg $ grep -rl "/internal" .
[tconn@Borg9 14:17]
~/Documents/code/git/moby/moby/pkg $ 
```
Same with Grafana's [pkg](https://github.com/grafana/grafana/tree/master/pkg) directory